src/app.py

```diff
@@ -107,24 +148,29 @@ def instantiate_file_builder(factory_definition,
 
     Returns
     -------
-    File_Builder
-        Instantiated file builder as defined by the file builder
-        configurations, as per the user specified output type of
-        the given object configuration
+    FileBuilder
```

Comment:
I don't think "as specified in the domain object configuration of the user config" really adds any value to this docstring